src/main/java/echoquery/sql/SchemaInferrer.java

```diff
@@ -77,26 +115,57 @@ private void adjustTableToForeignKeys(String table, DatabaseMetaData md)
     }
   }
 
-  private void adjustColumnToTable(String table, DatabaseMetaData md)
-      throws SQLException {
-    ResultSet columns = md.getColumns(null, null, table, null);
-    while (columns.next()) {
-      String col = columns.getString("COLUMN_NAME");
-      Set<String> previousSet =
-          columnToTable.getOrDefault(col, new HashSet<String>());
-      previousSet.add(table);
-      columnToTable.put(col, previousSet);
+  /**
+   * @param table The table that we want to aggregate over / select from.
+   * @param column The column that were filtering on.
+   * @return A JoinRecipe that can create the table to filter one and select
+   *    from.
+   */
+  public JoinRecipe infer(
```

Comment:
Things were just moved around and I just fixed some style, the only real change of this class is that I added getters for columnToTable and tables